### DISCUSSION TITLE
Overhaul of observation fields

### DISCUSSION THREAD

**Discussion by ezwelty** [falling-fruit/falling-fruit-web#930] [Ideas]
`observation.fruiting_status` currently only accepts the values ["Flowers", "Unripe fruit", "Ripe fruit"] but this isn't easily mapped to a binary in/out-of-season. Some plants have edible flowers, others edible fruit, others are used for other things. Still, there's value in tracking the flower-fruit sequence – e.g. for predicting fruiting or frost susceptibility, for pollinator forage). So the following has been suggested:

**Type** – *required if more than one at that location*
  - Apple [Malus sp.]
  - Common fig [Ficus caricus]
  - ...

**Status** – *optional*
  - Bare
  - Leaves only
  - Flowers
  - Unripe fruit
  - Ripe fruit
  - Fallen (absent) fruit

**In-season** – *optional*
  - No
  - Yes *selected automatically if "Ripe fruit" selected above*

**Parts in-season** – *if in-season, and type is a plant*
***Could this list be simplified?***
  - Fruit (including aril "false-fruits" like lychees, and vegetables that are fruits)
  - Flower (including buds, petals, pollen, nectar)
  - Seed or Seedpod (including nuts, cereals, beans, okra, green beans)
  - Root (including bulbs, rhizomes, tubers, corms)
  - Stem (including young shoots)
  - Leaf
  - Bark (including inner bark)
  - Sap (including sap from trees and manna from plants)

**Amount harvested** – *if in-season*
  - By volume: [ ] [ unit ]
  - By weight: [ ] [ unit ]

**Comment by ezwelty** [comment]
A user suggested "Fallen (absent) fruit" could instead be the more general "Past ripe". Once the fruit are absent, we're either back to "bare" or "leaves only".

**Comment by wbazant** [comment]
How about a fourth fruiting status, "No flowers or fruit"? That completes the menu, and captures both no more fruit for someone who found all their cherries taken by birds and not applicable for someone who tried to put fruiting status for a dumpster and got confused about what to put in it.

Very easy to implement (I think you just need to relax a database constraint and I'll add the extra form value). The list going up to five choices is still manageable. Focus on serving a broad audience is retained.

Being in season can stay as a curated summary on the location level. We can add a hint: if there are a few datapoints from reviews, say three fruiting status reviews, invite the user to fill out the season.
